def raise_exception_msg():
  raise NameError()
  return raise_exception_msg